commit 9ba734396909a002068076a5595b4f75de06b73f (refs/changes/99/4399/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-06-22T10:38:11+01:00 (1 year, 4 months ago)
    
    MANTA-3781 manta-mackerel build asks for tar when it wants gtar
    Reviewed by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 00f283a..3ad09ea 100644
--- a/Makefile
+++ b/Makefile
@@ -119,7 +119,7 @@ publish: release
 
 .PHONY: assets
 assets: $(NODEUNIT)
-	tar -zcf $(ROOT)/assets/node_modules.tar node_modules build/node/bin/node
+	gtar -zcf $(ROOT)/assets/node_modules.tar node_modules build/node/bin/node
 
 
 include ./tools/mk/Makefile.deps
